apps/frontend/src/components/TimeCardPage/SubmitCard.tsx

```diff
@@ -32,6 +35,11 @@ export default function SubmitCard(props: submitCardProps) {
    *   any associate, supervisor, or admin that was viewing the timesheet.
    */
   const [cardState, setCardState] = useState(TimesheetStatus.UNSUBMITTED);
+  const [submitDisabled, setSubmitDisabled] = useState(props.submitDisabled);
```

Comment:
Does this need to be a state variable is it's already being passed in through the props? I think this line can probably be removed if we are choosing to control the enabled/disabled state through the main Timesheet.tsx file